src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,17 @@
 import React from 'react';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
+import './CommentList.scss';
+import { CommentInfo } from '../CommentInfo';
+import { Comment } from '../../types/comment';
+
+type Props = {
+  comments: Comment[];
+};
+
+export const CommentList: React.FC<Props> = ({ comments }) => (
+  <ul className="CommentList">
+    {comments.map(comment => (
+      <CommentInfo comment={comment} key={comment.id} />
+    ))}
+  </ul>
```

Comment:
better to write `li` tag in the same component as `ul`

Reply:
ok